short-domain-name-service/src/main/java/com/scdtchina/config/SystemConfig.java

```diff
@@ -0,0 +1,42 @@
+package com.scdtchina.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * @author hai.yuan
+ * 系统配置类
+ */
+@Configuration
+@ConfigurationProperties(prefix = "system")
+public class SystemConfig {
+
+    /**
+     * 系统最大容量
+     */
+    private Long maxKey = 218340105584895L;

```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- ce70576b-8804-4d36-a131-84f1b200b094 -->